docs/learn-governance.md

```diff
@@ -167,9 +167,9 @@ In contrast, when it has 75% turnout, the tally of "aye" votes has to reach 54%,
 
 Suppose there is a unanimous proposal proposed by the council, ``Negative Turnout Bias`` would be used, so that means the proposal is passed by default. Hence, more token holders have to participate in voting to prevent it from passing if they do not like this proposal.
 
-Referring to the above image, when the referenda only has 25% turnout, the tally of "nay" votes has to reach 34% for it to reject.
+Referring to the above image, when the referenda only has 25% turnout, the tally of "aye" votes has to reach 34% for it to accept.
```

Comment:
I think the image needs to be fixed and labeled better, it does not make sense to me at all.

But I think this edit isn't right. The proposal passes by default, so people in this scenario are fighting to reject it. This should say:

```suggestion
Referring to the above image, when the referendum only has 25% turnout, the tally of "nay" votes must not exceed 66% or the referendum is rejected. When the turnout is 75%, the tally of "nay" votes is tolerated until 54%, anything beyond and the proposal fails.
```

@laboon does this sound about right?

Reply:
I think it's correct, but it's rather confusingly worded.  I'd keep it as aye votes instead of changing it to nay votes.

Reply:
Am I misunderstanding that even with zero aye votes the proposal would pass?

Reply:
Let me rephrase - your correction is correct but confusingly worded.  Can you reword your change so that it still references the "aye" votes and not the "nay" votes? 

With zero aye votes, no vote should ever be able to pass - this would mean either 100% against (which will always reject) or no voters at all (which, at least with the current code, means automatic rejection - see https://github.com/paritytech/substrate/blob/6611308111143362e210a0e72f302aab534f98f7/srml/democracy/src/vote_threshold.rs#L82)